assembl/graphql/schema.py

```diff
@@ -662,6 +722,34 @@ def mutate(root, args, context, info):
             for attr, value in kwargs.items():
                 setattr(thematic, attr, value)
             db = thematic.db
+
+            # add uploaded image as an attachment to the idea
+            image = args.get('image')
+            if image is not None:
+                filename = os.path.basename(context.POST[image].filename)
+                mime_type = mimetypes.guess_type(filename)[0]
+                uploaded_file = context.POST[image].file
+                uploaded_file.seek(0)
+                data = uploaded_file.read()
+                document = models.File(
+                    discussion=discussion,
+                    mime_type=mime_type,
+                    title=filename,
+                    data=data)
+                # if there is already an attachment, remove it (but the old
+                # image stays on the server)
+                if thematic.attachments:
+                    thematic.attachments.remove(thematic.attachments[0])
```

Comment:
I remove now the image as well, we can revisit it when we have an images gallery in the future.